public/app/features/search/types.ts

```diff
@@ -0,0 +1,54 @@
+export interface DashboardSection {
+  id: number;
+  uid?: string;
+  title: string;
+  expanded: boolean;
+  url: string;
+  icon: string;
+  score: number;
+  hideHeader?: boolean;
+  checked: boolean;
+  items: DashboardSectionItem[];
+  toggle?: (section: DashboardSection) => Promise<DashboardSection>;
+  selected?: boolean;
+}
+
+export interface DashboardSectionItem {
```

Comment:
this looks the same as https://github.com/grafana/grafana/blob/master/public/app/types/search.ts#L6 would be good to try to combine them if possible

Reply:
Hmm, I think it'd make sense to have all these types inside the `features/search`, but that would involve fixing all the imports/references to the previous types so maybe do that at a later point when we're starting to remove the old search components?

Reply:
yes, sounds like a plan